app/graphql/types/mutation_types.rb

```diff
@@ -5,5 +5,7 @@ module Types
   class MutationTypes < BaseObject
     # Plates
     field :create_plate_with_ont_samples, mutation: Mutations::CreatePlateWithOntSamplesMutation
+
+    field :poolSamples, mutation: Mutations::PoolSamplesMutation
```

Comment:
The field should be snake_case and is the name of the method you supply when you request the mutation

Reply:
I think it should also be more descriptive of what the mutation does and the class name for the mutation should match as well.  Something like `:create_ont_library_by_pool_samples`